DSNP-Wallet/Source/Modules/TestViewController.swift

```diff
@@ -65,64 +71,89 @@ extension TestViewController {
     }
     
     @objc func tappedBtn(selector: UIButton?) {
-        let completion: ExtrinsicSubmitClosure = { result in
-            var text = ""
-            
-            switch result {
-            case .success(let result):
-                text = result
-                
-            case .failure(let error):
-                text = "\(error)"
-            }
-            
-            let alert = UIAlertController(title: "Result", message: text, preferredStyle: .alert)
-            alert.addAction(UIAlertAction(title: "Ok", style: .cancel, handler: nil))
-            self.present(alert, animated: true)
-        }
-        
         let primaryMnemonicInput = primaryTextField.text?.isEmpty ?? true ? "quote grocery buzz staff merit patch outdoor depth eight raw rubber once" : primaryTextField.text ?? ""
         let primaryUser = User(mnemonic: primaryMnemonicInput)
         
         let secondaryMnemonicInput = secondaryTextField.text?.isEmpty ?? true ? "wink settle steak second tuition whale question must honey fossil spider melt" : secondaryTextField.text ?? ""
```

Comment:
why do we need 2 mnemonics again?

Reply:
Secondary mnemonic is specifically for addPublicKeyToMsa(). The extrinsic requires signing from two address: i) the original address where the MSA came from (say custodial wallet), and ii) the new address which the MSA will be assigned to (in our case the wallet). It's redundant, but the way that the blockchain framed that extrinsic for reasons I don't remember